style.css

```diff
@@ -0,0 +1,721 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+
+  /* padding: 0; */
```

Comment:
- [ ] Kindly make sure to delete this line too and all other commented-out lines of code to make your code clean :recycle: :